kubeflow/common/iap-ingress/base/swagger_template.yaml

```diff
@@ -0,0 +1,64 @@
+swagger: "2.0"
```

Comment:
NIT: Consider adding a comment or README to explain where is this file coming from.

Reply:
This is a generic template. It came from the legacy [cloud-endpoints-controller](https://github.com/danisla/cloud-endpoints-controller), which is no longer maintained. I am afraid that additional reference might confuse people. Let me know what you think.

Reply:
That is true that it might be risky to refer to non-maintained repository. However, we are still using such legacy template within our solution. We should still document where this template is coming from, so other maintainer knows that where to find it and the current state is `repository not-maintained`. In situations like this, more documentation is better than less documentation. 

Would you like to take the following actions?
1. Create a README file within `kubeflow/common/iap-ingress`.
2. Explain where `swagger_template.yaml` and `setup_cloudendpoints.sh` are originating from
3. Explain that the `cloud-endpoints-controller` repository is no longer maintained.
4. We can also list potential improvement where we can move away from any legacy logic. I found a few materials below. Feel free to decide whether to include them:

openapi.yaml sample: https://github.com/GoogleCloudPlatform/endpoints-samples/blob/master/k8s/openapi.yaml 
Single app example using IAP, GKE, Cloud endpoint: https://github.com/salrashid123/iap_endpoints_app
IAP ingress controller (might be deprecated as well): https://github.com/danisla/iapingress-controller/tree/master